webpack/JobInvocationDetail/JobInvocationDetail.scss

```diff
@@ -38,4 +38,8 @@
     height: $chart_size;
   }
 }
+
+.job-invocation-host-table-page-section {
+  padding-top: 0px;
+}
```

Comment:
There is no element with this classname.
Also can the table sides have no padding like in the hosts page?